pages/search/index.tsx

```diff
@@ -106,67 +111,102 @@ export default function Search({
     searchResultsHeadingRef?.current?.focus()
   }, [isLoading, isFreshSortByQuery])
 
+  const displayFilters = !!aggs?.filter((agg: Aggregation) => agg.values.length)
+    .length
+
   return (
     <>
       <RCHead metadataTitle={metadataTitle} />
       <Layout
         bannerNotification={bannerNotification}
         searchAggregations={aggs}
+        searchResultsCount={totalResults}
         isAuthenticated={isAuthenticated}
         activePage="search"
         sidebar={
-          <>
-            {totalResults > 0 ? (
-              <SearchResultsSort
-                searchParams={searchParams}
-                handleSortChange={handleSortChange}
-                // We have to render the sort select twice and toggle which is shown at the desktop breakpoint, since
-                // the design has it appearing in the sidebar on desktop and in the main content on mobile.
-                // Using inline styles to do this for now since using useNYPLBreakpoints had a visible lag.
-                // TODO: Extend the Layout component to receive a prop that contains content to be shown below the
-                //  main header, which will include the search results heading and the sort select, which would allow us
-                //  to only render the sort select once.
-                display={{
-                  base: "none",
-                  md: "block",
-                }}
-              />
-            ) : null}
-            {isLoading ? (
-              <SkeletonLoader showImage={false} />
-            ) : (
+          totalResults > 0 ? (
+            <Box display={{ base: "none", md: "block" }} width="100%" pb="l">
+              {displayFilters && (
+                <Card
+                  id="filter-sidebar-container"
+                  backgroundColor="ui.bg.default"
+                  p="s"
+                  borderRadius="8px"
+                  mb="s"
+                >
+                  <CardHeading size="heading6" id="filter-results-heading">
+                    Filter results
+                  </CardHeading>
+                  <CardContent>
+                    <SearchFilters aggregations={aggs} />
+                  </CardContent>
+                </Card>
+              )}
               <DRBContainer
                 drbResults={drbResults}
                 totalWorks={drbResponse?.totalWorks}
                 searchParams={searchParams}
               />
-            )}
-          </>
+            </Box>
+          ) : isLoading ? (
+            <SkeletonLoader showImage={false} />
+          ) : null
         }
       >
         {totalResults ? (
-          <>
+          <Box
+            sx={{
+              ["@media screen and (min-width: 600px)"]: {
```

Comment:
Can you use the design token medium break point here instead of the media query?